locale/en_x_pirate.po

```diff
@@ -7,15 +7,17 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-04-26 14:46-0500\n"
-"PO-Revision-Date: 2018-07-27 11:55-0700\n"
-"Last-Translator: \n"
-"Language-Team: @sommersoft, @MrCertainly\n"
+"POT-Creation-Date: 2020-03-20 17:57-0500\n"
+"PO-Revision-Date: 2020-03-30 22:11+0000\n"
```

Comment:
In https://github.com/adafruit/circuitpython/pull/2799/commits/b0383f4aff0259b9d3fdcf23aa736fc973b1eff6, I removed `PO-Revision-Date` from `circuitpython.pot` template and from all the `.po` files, and it seems to have worked. I'm not sure if it's going to be restored by some .po tools, but it should help fix some unnecessary merge diffs. I don't see any reason for that field since the file is under source control and its history can be tracked.

Reply:
I will look into whether this can be controlled in weblate.

Reply:
if necessary, the `make` targets for translation could use  `sed` to remove those lines.

Reply:
I think if it's not in the `.pot`, it may not show up in the `.po` files

Reply:
weblate's using its own internal methods to update these files, not our usual tool.  They are not keen to change it, even though others have suggested omitting this metadata: https://github.com/WeblateOrg/weblate/issues/55#issuecomment-603208399

Reply:
I think we've decide to live with this for now